packages/tldraw/src/test/ZoomTool.test.ts

```diff
@@ -209,4 +209,6 @@ describe('TLSelectTool.Zooming', () => {
 		})
 		editor.expectToBeIn('zoom.idle')
 	})
+
+	it("Correctly doesn't zoom in when the zoom lock is deactivated", () => {})
```

Comment:
Is this test meant to just be empty?

Reply:
LGTM besides this comment right here. Is it meant to be empty?

Reply:
yeah i guess ur right